fixtures/dom/src/components/fixtures/home.js

```diff
@@ -0,0 +1,90 @@
+const React = window.React;
+
+export default function Home() {
+  return (
+    <main>
+      <h1>DOM Test Fixtures</h1>
+      <p>
+        Use this site to test browser quirks and other behavior that can not be
+        captured through unit tests.
+      </p>
+      <section>
+        <h2>Tested Browsers</h2>
+        <table>
+          <thead>
+            <tr>
+              <th>Browser</th>
+              <th>Versions</th>
+              <th>Why/Notes</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Chrome - Desktop</td>
+              <td>49, Latest</td>
+              <td>49 is the last release for Windows XP</td>
+            </tr>
+            <tr>
+              <td>Chrome - Android</td>
+              <td>Latest</td>
+              <td>N/A</td>
+            </tr>
+            <tr>
+              <td>Firefox Desktop</td>
+              <td>
+                <a href="https://www.mozilla.org/en-US/firefox/organizations/">
+                  ESR
+                </a>, Latest
+              </td>
+              <td>
+                The long term support release is used by many institutions
+              </td>
+            </tr>
+            <tr>
+              <td>Internet Explorer</td>
+              <td>9, 10, 11</td>
+              <td>N/A</td>
+            </tr>
+            <tr>
+              <td>Microsoft Edge</td>
+              <td>14, Latest</td>
+              <td>N/A</td>
+            </tr>
+            <tr>
+              <td>Safari - Desktop</td>
+              <td>7, Latest</td>
+              <td>N/A</td>
+            </tr>
+            <tr>
+              <td>Safari - iOS</td>
+              <td>7, Latest</td>
+              <td>N/A</td>
+            </tr>
+          </tbody>
+        </table>
+      </section>
+      <section>
+        <h2>How do I test browsers I don't have access to?</h2>
```

Comment:
I like that section a lot! Maybe we can also list some free options if money is an issue. For example:

- Microsoft provides the option to download free virtual machines with various Internet Explorer versions: https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/
- You can download a legacy version of Chromium by looking up the base position in https://omahaproxy.appspot.com/ and using that number in https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html. 
  - Maybe that's a little to complicated? We can skip the first step and hardcode `369907` as the base position?
  - Alternatively we can link to https://www.chromium.org/getting-involved/download-chromium
- Firefox ESR versions can be downloaded here: https://www.mozilla.org/en-US/firefox/organizations/


I guess that's it. For Safari you'd need a VM to an old macOS version which will require running macOS as well.

Reply:
Those are all great ideas! I'll update this section with these recommendations.